misc/python/materialize/feature_benchmark/benchmark.py

```diff
@@ -127,14 +127,14 @@ def append(self, comparison: Comparator) -> None:
 
     def dump(self) -> None:
         print(
-            f"{'NAME':<25} | {'THIS':^11} | {'OTHER':^11} | {'Regression?':^13} | 'THIS' is:"
+            f"{'NAME':<35} | {'THIS':^11} | {'OTHER':^11} | {'Regression?':^13} | 'THIS' is:"
```

Comment:
Could use a library like https://github.com/astanin/python-tabulate or iterate through all the entries initially to find the longest entry, then set that as length here.

Reply:
he he nothing less than https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html should be used! Are we in the AI century or what? :-)

Seriously, let me put your suggestion as a good-first-issue -- that dumper was always giving me trouble.